linux/network_device_ops/pvrdma_client_network_device_ops.yml

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Perform network device operations on client VM
+#
+- name: "Test block on PVRDMA client VM"
+  block:
+    - name: "Set facts to perform network device operations on client VM"
+      ansible.builtin.set_fact:
+        vm_name: "{{ pvrdma_client_vm_name }}"
+        current_test_log_folder: "{{ pvrdma_client_test_log_folder }}"
+        pvrdma_client_vm_ready: false
+        pvrdma_client_test_success: false
```

Comment:
This parameter `pvrdma_client_test_success` can be replaced by setting to 'fail' in 'rescue' part directly.

Reply:
update.d